src/web/services/feedback-sessions.service.ts

```diff
@@ -314,7 +314,7 @@ export class FeedbackSessionsService {
   /**
    * Retrieves the results for a feedback session.
    */
-  getFeedbackSessionsResult(queryParams: {
+  getFeedbackSessionResults(queryParams: {
```

Comment:
Not sure whether the naming for this was intentional, can wait for comment from senior dev